daffodil-cli/src/test/scala/org/apache/daffodil/cli/cliTest/TestCLIParsing.scala

```diff
@@ -220,7 +220,7 @@ class TestCLIParsing {
     }(ExitCode.Success)
   }
 
-  @Test def test_1317_IBMCompatibility_ABC_test_ibm_abc_cli(): Unit = {
+  @Test def test_IBMCompatibility_ABC_test_ibm_abc_cli(): Unit = {
```

Comment:
Could rename to `test_CLI_IBMCompatibility_ABC_test_ibm_abc` to be more similar to rest of test case names, any disagreement?